apps/dashboard/app/helpers/application_helper.rb

```diff
@@ -23,11 +23,11 @@ def restart_url
   # @param role [String] app role i.e. "vdi", "shell", etc.
   # @param data [Hash] data parameter for the link_to helper.
   # @return nil if url not set or the HTML string for the bootstrap nav link
-  def nav_link(title, icon, url, new_tab: false, role: nil, data: nil)
+  def nav_link(title, icon, url, new_tab: false, data: nil)
```

Comment:
all nav links have the same role, so i'm removing this bit (nobody is using it anyhow).